Mathlib/RingTheory/MvPolynomial/Homogeneous.lean

```diff
@@ -42,6 +42,10 @@ TODO
 /-- The degree of a monomial. -/
 def degree (d : σ →₀ ℕ) := ∑ i ∈ d.support, d i
 
+lemma degree_eq_zero_iff (d : σ →₀ ℕ) : degree d = 0 ↔ d = 0 := by
```

Comment:
Does it make sense to have this as a `simp` lemma?

Reply:
It would make sense, but does not seem necessary. (We discussed that briefly with Filippo)